moveit_ros/planning/moveit_cpp/src/moveit_cpp.cpp

```diff
@@ -55,11 +55,8 @@ MoveItCpp::MoveItCpp(const rclcpp::Node::SharedPtr& node, const std::shared_ptr<
 
 MoveItCpp::MoveItCpp(const rclcpp::Node::SharedPtr& node, const Options& options,
```

Comment:
`tf_buffer` is not used anymore so we should provide a constructor without it. Technically, we should deprecate all constructors that used `tf_buffer` instead of removing them immediately, also.